lib/srv/desktop/rdp/rdpclient/client.go

```diff
@@ -253,6 +253,7 @@ func (c *Client) connect(ctx context.Context) error {
 		C.uint16_t(c.clientHeight),
 		C.bool(c.cfg.AllowClipboard),
 		C.bool(c.cfg.AllowDirectorySharing),
+		C.bool(c.cfg.ShowDesktopWallpaper),
```

Comment:
Instead of continuing to add a bunch of arguments to the connect func, maybe we should just have a u32 with flags for all these bools. 

Thoughts?

Reply:
Seems like overkill to concern ourselves with being that space-efficient here. It is a bit disorganized here though, I cleaned it up by shoving everything into a struct: 8066e302c892d0c9754612bd8501b4818ae44c00

Reply:
The concern was not so much about space efficiency, just a reasonable number of arguments. If it was pure Go I would have suggested a struct, but for calls via a C interface nothing is simpler than a fixed size integer, so that's why I suggested flags.

What you have is okay too, it's just one more type we have to publish across the C boundary.